docs/migration/6_0/deleted-api.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "apiElementName": "SomeService",
+    "entryPoint": "@spartacus/feature/core",
+    "migrationComment": "Use class OtherService instead.  (note: this example won't appear in the migration material because the SomeService class does not exist)"
+  },
+  {
+    "apiElementName": "AwesomeClass",
+    "entryPoint": "@spartacus/feature/components",
+    "migrationComment": "This class is not needed anymore.  Things are done in this way instead.  (note: this example won't appear in the migration material because the AwesomeClass class does not exist)"
+  }
+]
```

Comment:
> some cleanup from the escaped single quote is needed, which appends the 'backslash \' as shown in the screenshots

The quotes are escaped because the string is delimited by the same kind of quotes.  It's the choice of the api used to stringily the object array.